portfolio/src/test/java/com/google/sps/NameTest.java

```diff
@@ -0,0 +1,47 @@
+package com.google.sps.servlets;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.ServletConfig;
+import javax.servlet.ServletOutputStream;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mockito;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(JUnit4.class)
+public final class NameTest extends Mockito {
+  private final DataServlet servlet = new DataServlet();
+
+  @Mock
+  private HttpServletRequest request;
```

Comment:
probably want to make these `private final` variables since they will be the only request/response/etc. variables you will use throughout the entire class